source/php/Component/Fileinput/fileinput.blade.php

```diff
@@ -17,25 +17,19 @@ class="{{ $baseClass }}__input "
         id="fs_{{ $id }}"
         accept="{{ $accept }}"
         {{ $multiple ? 'multiple' : '' }}
-        @if($required)
-            required
-            data-required="1"
-            aria-required="true"
-        @endif
     />
 
-    <label for="fs_{{ $id }}" class="c-button c-button__filled c-button__filled--primary c-button--md {{ $baseClass }}__label">
```

Comment:
Isen't it pssible to use button component?

Reply:
I believe label is used to allow the user to click the text connecting it to the input field. Since the input has not height or width the label is the only way to simulate a click.